### DISCUSSION TITLE
Limit which users are allowed to create new projects

### DISCUSSION THREAD

**Discussion by mvarblow** [NationalSecurityAgency/skills-service#2784] [Q&A]
Is it possible to limit which users have permission to create new projects? We'd like all our users to be able to participate in the training, but don't want all users to be able to create new projects or otherwise add new training content. I can't see a way to do this with the latest version (3.0.2). Thanks!

**Comment by sudo-may** [comment]
Hi @mvarblow,  Thank you for reaching out. You are correct, currently the SkillTree platform does not stop users from visiting the administrative portion of the Dashboard and potentially creating new projects (training profiles). It does seem like it would make sense to allow root Dashboard users to configure a list of users that are allowed to use/visit the admin portion of the Dashboard - this way admin  pages can also be hidden from the rest of the users. I'd encourage to file a new ticket for consideration. Thank you very much!

**Reply by mvarblow** [reply]
Thanks for the quick response! I've added https://github.com/NationalSecurityAgency/skills-service/issues/2785
